src/main/drivers/accgyro_spi_mpu6500.c

```diff
@@ -99,10 +99,14 @@ static bool mpu6500Detect(void)
     uint8_t tmp;
 
     mpu6500ReadRegister(MPU6500_RA_WHOAMI, &tmp, 1);
+#ifdef COLIBRI_RACE
```

Comment:
I am not sure why, because if it checking the WHOAMI command will return fail.
That's why i make it return true anyway.